river/command/output.zig

```diff
@@ -52,7 +52,20 @@ pub fn sendToOutput(
     _: *?[]const u8,
 ) Error!void {
     if (args.len < 2) return Error.NotEnoughArguments;
-    if (args.len > 2) return Error.TooManyArguments;
+    const result = flags.parser([:0]const u8, &.{
```

Comment:
You also need to import `flags` before you use it.

Reply:
Done.